tools/src/bin/output-file.rs

```diff
@@ -335,22 +335,58 @@ fn main() {
                     accel_key: None,
                     copyable: false,
                 });
-                if let Some(ref hg_root) = tree_config.paths.hg_root {
+
+                let gh_log_link = tree_config
+                    .paths
+                    .github_repo
+                    .as_ref()
+                    .map(|gh_root| format!("{}/commits/{}/{}", gh_root, oid, encoded_path));
```

Comment:
Here I've pointed to the current commit (`oid`), but actually to match the existing Mercurial behavior that should point to `HEAD`.

Reply:
Yeah, I think we probably want the branch tip.  So I guess we potentially need to pull the branch name that's checked out from the "git" dir?  We could of course put it in the config, but that's sorta currently implicit from the disk state...

Reply:
The Mercurial version just uses `tip`, I think that's the Git equivalent for `HEAD`?